judge/judgedaemon.main.php

```diff
@@ -872,6 +877,50 @@ function fetch_executable_internal(
         continue;
     }
 
+    if ($type == 'output_visualization') {
+        if ($lastWorkdir !== null) {
+            cleanup_judging($lastWorkdir);
+            $lastWorkdir = null;
+        }
+        foreach ($row as $judgeTask) {
+            if (isset($judgeTask['output_visualizer_script_id'])) {
+                // Visualization of output which this host judged requested.
+                $run_config = dj_json_decode($judgeTask['run_config']);
+                $tmpfile = tempnam(TMPDIR, 'output_visualization_');
+                [$runpath, $error] = fetch_executable_internal(
+                    $workdirpath,
+                    'output_visualization',
+                    $judgeTask['output_visualizer_script_id'],
+                    $run_config['hash']
+                );
+                if (isset($error)) {
+                    // FIXME
```

Comment:
Fix me 😛

Reply:
Loosely based on what we do for the debug tasks, I'll see if I can merge the shared parts and possibly fix this one.